API_Extended_Document/persistence_unit/PersistenceUnit.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python3
+# coding: utf8
+
+from contextlib import contextmanager
+
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+
+from util.db_config import get_db_info
+
+db_string = get_db_info("util/config.yml")
+engine = create_engine(db_string)
+Session = sessionmaker(engine)
+
+"""
+makeATransaction is a function that allows to re-use the same code
```

Comment:
After some discussion with LTA, it appears that using the side effects of a generator in order to re-use a typical session context is (within the context of this code) not the optimal technical manner of apply the DRY principle.
Using classic decorators seems a more classic and ad-hoc way.